status/versions.go

```diff
@@ -26,10 +26,10 @@ var (
 	//   redhat@redhat-pulp.abc.xyz.redhat.com:8888/openshift-istio-tech-preview-0.1.0-1-3a136c90ec5e308f236e0d7ebb5c4c5e405217f4-Custom
 	// Example Istio snapshot version is:
 	//   root@f72e3d3ef3c2-docker.io/istio-release-1.0-20180927-21-10-cbe9c05c470ec1924f7bcf02334b183e7e6175cb-Clean
-	maistraProductVersionExpr = regexp.MustCompile("maistra-([0-9]+\\.[0-9]+\\.[0-9]+)")
-	maistraProjectVersionExpr = regexp.MustCompile("openshift-istio.*-([0-9]+\\.[0-9]+\\.[0-9]+)")
-	istioVersionExpr          = regexp.MustCompile("([0-9]+\\.[0-9]+\\.[0-9]+)")
-	istioSnapshotVersionExpr  = regexp.MustCompile("istio-release-([0-9]+\\.[0-9]+)(-[0-9]{8})")
+	maistraProductVersionExpr = regexp.MustCompile(`maistra-([0-9]+\.[0-9]+\.[0-9]+)`)
```

Comment:
What's the point of this? Strings should be with ``"`` to make any sense to the reader. Is the linter causing this sort of sillyness?

Reply:
back-quoting strings allows to avoid escaping with "\\", it often makes regex easier to read. And yes that was reported by one of the linters